backend/src/v2/metadata/client.go

```diff
@@ -52,6 +52,7 @@ type ExecutionType string
 const (
 	ContainerExecutionTypeName ExecutionType = "system.ContainerExecution"
 	DagExecutionTypeName       ExecutionType = "system.DAGExecution"
+	MaxClientGRPCMessageSize                 = 100 * 1024 * 1024
```

Comment:
Hi @haoxins, can you please give us a bit context on how did you hit this error? Given this is the metadata client, I imagine the request payload should be very small. Even if you raise this limit, I suspect there could be other limits in our system.

Reply:
I met this error by pass a file (`OutputPath`) from one component into next component which the size is `6mb`.